src/Dialect/ONNX/ONNXOps.td.inc

```diff
@@ -821,7 +821,7 @@ def ONNXConvTransposeOp:ONNX_Op<"ConvTranspose",
   "output_shape can also be explicitly specified in which case pads values are auto generated using these equations:"
   ""
   "  total_padding[i] = stride[i] * (input_size[i] - 1) + output_padding[i] + ((kernel_shape[i] - 1) * dilations[i] + 1) - output_shape[i]"
-  "  If (auto_pads != SAME_UPPER): pads[start_i] = total_padding[i]/2; pads[end_i] = total_padding[i] - (total_padding[i]/2)"
+  "  If (auto_pads == SAME_UPPER): pads[start_i] = total_padding[i]/2; pads[end_i] = total_padding[i] - (total_padding[i]/2)"
```

Comment:
Could you comment on the source of all these text changes in the ONNXOPS.td.inc?